src/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { IDeserializedSrn } from './types';
+
+export function deserializeSrn(srn: string): IDeserializedSrn {
+  const [service, org, project, ...uriParts] = srn.split('/');
+
+  const uri = uriParts.length ? `/${uriParts.join('/')}` : '';
+
+  return {
+    service,
+    org,
+    project,
+    uri,
+  };
+}
```

Comment:
I don't think this is needed anymore. `ui-kit` doesn't need to know what an "srn" is